src/components/attendee/Button/Button.tsx

```diff
@@ -28,16 +27,16 @@ export const Button: React.FC<ButtonProps> = ({
   transparent,
   forwardRef,
   border,
-  borderRadius,
-  margin,
+  rounded = true,
```

Comment:
This might need tweaking I guess. `unrounded` works as both missing (`undefined`) and present (`true`) :/ Either way, we can revisit some DX changes later on, see what worked, what didn't.

Reply:
changed the prop: https://github.com/sparkletown/sparkle/pull/2953/commits/d110155d478fa8fcec0c3dce7443ed2d4b871a01